src/braket/jobs/_entry_point_template.py

```diff
@@ -1,8 +1,16 @@
 run_entry_point = """
 import cloudpickle
-from braket.jobs import save_job_result
+import os
+from braket.jobs import get_results_dir, save_job_result
 from braket.jobs_data import PersistedJobDataFormat
 
+
+# set working directory to results dir
+os.chdir(get_results_dir())
+
```

Comment:
is it also possible to mkdir if they don't exist? Example
```
np.save("my_dir/file.npy")
```
should this automatically create the dir "my_dir"?

Reply:
I don't know the answer to that, but that's squarely a numpy functionality question. We shouldn't be inspecting path arguments to check whether they map to existing directories etc